sections/07_conclusions.md

```diff
@@ -9,19 +9,18 @@ problem being addressed, we conclude that deep learning has not *yet* realized
 its transformative potential or induced a strategic inflection point.  Despite
 its dominance over competing machine learning approaches in many of the areas
 reviewed here and quantitative improvements in predictive performance, deep
-learning has not yet definitively "solved" those problems.
+learning has not yet definitively "solved" these problems.
 
 As an analogy, consider recent progress in conversational speech recognition.
-Since 2009 there have been drastic performance improvements, with error rates
+Since 2009 there have been drastic performance improvements with error rates
 dropping from more than 20% to less than 6% [@tag:Speech_recognition] and
 finally approaching or exceeding human performance in the past year
 [@arxiv:1610.05256 @arxiv:1703.02136]. The phenomenal improvements on benchmark
 datasets are undeniable, but greatly reducing the error rate on these benchmarks did not
 fundamentally transform the domain.  Widespread adoption of conversational
-speech technologies will require not only improvements over baseline methods but
-truly solving the problem, in this case exceeding human-level performance, as
```

Comment:
This phrasing was awkward, but I wrote it with such complexity to emphasize "solving the problem".  In speech recognition, human performance may be the goal line for solving the problem.  For many tasks we review, human performance isn't relevant though.  Is there a way to convey that more concisely?